### DISCUSSION TITLE
Plugins overview page?

### DISCUSSION THREAD

**Discussion by phiamo** [ionic-team/capacitor#3155] [General] [answered]
Hi guys thanks a lot for this amazing piece of software!!!!
Before the docs recently changed to finally lead me herebthere was a list with existing plugins 
Is there any chance to find that now? In any capacitor wiki or similar?

**Comment by imhoffd** [comment, marked as answer]
You can find the list in our git history: https://github.com/ionic-team/capacitor/blob/53c0c155d472d3e456cf584d29b0897455c06f25/site/docs-md/community/plugins.md

We won't be maintaining the old list anymore. Instead, we encourage you to create Community Plugin proposals (even if the plugin exists already): https://github.com/capacitor-community/proposals
